index.html

```diff
@@ -26,36 +26,191 @@ <h4>Rabaya Rabu</h4>
         <!-- headline section -->
```

Comment:
Kindly remove the below line in your readme file:
<img width="620" alt="image" src="https://user-images.githubusercontent.com/37116322/227625940-f328b4d9-f1a8-4ba9-8407-a51b1a0cd6f9.png">
 This note is a part of the guideline template and should not be part of your final readme